portable/MSVC-MingW/portmacro.h

```diff
@@ -184,8 +184,9 @@ void vPortExitCritical( void );
 #define portTASK_FUNCTION_PROTO( vFunction, pvParameters )    void vFunction( void * pvParameters )
 #define portTASK_FUNCTION( vFunction, pvParameters )          void vFunction( void * pvParameters )
 
-#define portINTERRUPT_YIELD    ( 0UL )
-#define portINTERRUPT_TICK     ( 1UL )
+#define portINTERRUPT_YIELD                        ( 0UL )
+#define portINTERRUPT_TICK                         ( 1UL )
+#define portINTERRUPT_APPLICATION_DEFINED_START    ( 2UL )
```

Comment:
What is the use of this config?

Reply:
This allows application writer to define interrupt.
For example, in cellular comm interface application
```c
#define portINTERRUPT_UART portINTERRUPT_APPLICATION_DEFINED_START    
```